content/kubermatic/main/installation/install-kkp-CE/add-seed-cluster-CE/_index.en.md

```diff
@@ -1,8 +1,10 @@
 +++
-title = "Add Seed Cluster for CE"
+title = "Add Seed Cluster to Kubermatic Kubernetes Platform (KKP) CE"
+linkTitle = "Add Seed Cluster to CE"
 date = 2018-08-09T12:07:15+02:00
 description = "Learn to add a new seed cluster to an existing KKP master cluster (Community Edition)"
 weight = 40
+enableToc = true
 +++
 
 This document describes how a new seed cluster can be added to an existing KKP master cluster.
```

Comment:
I'd add a note that this expects KKP to be already installed and link to the original installation document.